src/editions/transitioning-your-code-to-a-new-edition.md

```diff
@@ -112,3 +112,11 @@ the warnings, that's a bug!
 
 Once you're warning-free with this command you're good to go. Enjoy the new
 edition!
```

Comment:
I would move this new bit up over "Once you're warning-free with this command you're good to go. Enjoy the new edition!" so that "Enjoy the new edition!" is the last thing a user reads here.